grafana-plugin/src/pages/integration/OutgoingTab/ConnectedIntegrationsTable.tsx

```diff
@@ -84,11 +84,18 @@ const ConnectedIntegrationsTable: FC<ConnectedIntegrationsTableProps> = observer
                 <HorizontalGroup>
                   <Text type="secondary">Backsync</Text>
                   {tokenExists ? (
-                    <Tooltip content={<>Switch on to start sending data from other integrations</>}>
+                    <Tooltip
+                      content={
+                        <>
+                          Switch on to receive updates from ServiceNow. If disabled, Grafana OnCall will still send
+                          alerts to ServiceNow, but not receive updates back.
```

Comment:
Slight correction
- Switch on to receive updates from ServiceNow. If disabled, Grafana OnCall will still send alerts to ServiceNow, but **will** not receive **any** updates back.

Reply:
https://github.com/grafana/oncall/pull/4229/commits/41f1cba784fa9217e53a8a2269ebd6a3922f1ef2